x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-hipaa.adoc

```diff
@@ -179,9 +179,9 @@ The following is a third portion of the same `005010X222A2` `837` transaction se
 
 These two definitions apply to different instances of the `DTP` segment, as part of the 2300 Claim Information loop. In the transaction set structure these uses of the `DTP` segment occur essentially in the same position, matching two possible occurrences of a repeating DTP segment in the base X12 standard. Because the two uses of the segment are supplying different information, the HIPAA standard gives them different names and interprets the `DTP03` field in different ways.
```

Comment:
```suggestion
These two definitions apply to different instances of the `DTP` segment, as part of the 2300 Claim Information loop. In the transaction set structure, these uses of the `DTP` segment occur essentially in the same position, matching two possible occurrences of a repeating DTP segment in the base X12 standard. Because the two uses of the segment supply different information, the HIPAA standard gives them different names and interprets the `DTP03` field in different ways.
```